boot.js

```diff
@@ -130,12 +131,12 @@ function Boot (server, opts, done) {
   }
 
   this._doStart = null
-  const main = new Plugin(this, (s, opts, done) => {
+  const main = new Plugin(this, function root (s, opts, done) {
```

Comment:
Why did you remove the arrow function?

Reply:
So I get `root` as the name in the debug.